package.json

```diff
@@ -43,6 +43,8 @@
     "fs-extra": "0.18.4",
     "glob": "4.3.2",
     "html-to-text": "1.3.0",
+    "intl": "^1.0.0",
+    "intl-messageformat": "^1.1.0",
```

Comment:
Sorry, tiny nit-pick but can we please not have the `^`? We try to in everything to an exact version.